stablehlo/reference/Ops.cpp

```diff
@@ -40,6 +40,14 @@ std::vector<int64_t> permute(ArrayRef<int64_t> array, ArrayRef<int64_t> perm) {
 
 }  // namespace
 
+Tensor eval(AbsOp op, const Tensor &operand) {
+  Tensor result(op.getType());
+  for (auto it = result.index_begin(); it != result.index_end(); ++it) {
+    result.set(*it, abs(operand.get(*it)));
+  }
```

Comment:
Don't need curly braces for 1-liner. Though we don't have a coding guideline atm, we should follow the [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html) for now. Similarly in Element.cpp.